dev/tests/js/jasmine/tests/app/code/Magento/AdobeStockImageAdminUi/view/adminhtml/web/js/components/grid/column/overlay.test.js

```diff
@@ -0,0 +1,283 @@
+                        'response': 'success'
+                    }).promise();
+                });
+                testObj.ajaxFunction();
+                expect(handler.success).toHaveBeenCalled();
+                expect(typeof handler.success()).toBe('object');
+                expect(handler.success().length).toBe(2);
+            });
+
+            it('Check Ajax failure request', function() {
+                spyOn(handler, 'fail').and.callFake(function(e) {
+                    return 'Error Message';
+                });
+                spyOn($, 'ajax').and.callFake(function(e) {
+                    return $.Deferred().reject({
+                        'response': 'failure'
+                    }).promise();
+                });
+                testObj.ajaxFunction();
+                expect(handler.fail).toHaveBeenCalled();
+                expect(typeof handler.fail()).toBe('string');
+                expect(handler.fail()).toBe('Error Message');
+            });
+        });
+
+        describe('"getIds" method', function () {
+
+            it('Check for defined ', function () {
+                expect(overlayObj.hasOwnProperty('getIds')).toBeDefined();
+            });
+
+            it('Check method type', function () {
+                var type = typeof overlayObj.getIds;
+                expect(type).toEqual('function');
+            });
+
+            it('Check returned value type if method called with object argument', function () {
+
+                overlayObj.getIds = jasmine.createSpy().and.callFake(function () {
+                    return [1,2,3];
+                });
+
+                expect(typeof overlayObj.getIds()).toEqual('object');
+            });
+
+            it('Check getIds and return array', function () {
+
+                overlayObj.getIds = jasmine.createSpy().and.callFake(function () {
+                    return [1,2,3];
+                });
+
+                expect(overlayObj.getIds().length).toBe(3);
+            });
+
+        });
+
+        describe('"getStyles" method', function () {
+
+            it('Check for defined ', function () {
+                expect(overlayObj.hasOwnProperty('getStyles')).toBeDefined();
+            });
+
+            it('Check method type', function () {
+                var type = typeof overlayObj.getStyles;
+                expect(type).toEqual('function');
+            });
+
+            it('Check returned value type if method called with object argument', function () {
+
+                overlayObj.getStyles = jasmine.createSpy().and.callFake(function () {
+                    return {
+                        top: '100px'
+                    };
+                });
+
+                expect(typeof overlayObj.getStyles()).toEqual('object');
+            });
+
+            it('Check getStyles and return array', function () {
+
+                overlayObj.getStyles = jasmine.createSpy().and.callFake(function () {
```

Comment:
This test is not valuable as you are stubbing out the `getStyles` directly before testing it. Either don't stub the method you are testing, or remove this test.